From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@TwoUnder__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mausher). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess 
Please review and approve, thanks.

rmca14 commented:
@JennieHubbard 
Hi, Jennie. Can you please review and let me know if I should merge this PR? Barb is apparently OOF. Thanks.

JennieHubbard commented:
Approved!

rmca14 commented:
Separate mail sent to author re merge conflict.

angusrtaylor committed:
updated broken link

carolinacmoravia committed:
M12648: Case of ambiguity:  "activities on which email address"

Hello, @sharonlo101,
 
Translator has reported possible source content issue. It seems that "on which email address" means "in which the email address." This proposed file change avoids ambiguity.
 
Please, check this and help to merge if agreed. If not, please, let me know either if I should close this PR, if you would like me to fix it in another way within this PR or if you prefer to fix it in another PR. Could you indicate your PR number in that case so I can check the new version, please?

Many thanks in advance.

Commit:
PS Quickstart

Commit:
cred notes

Commit:
Removing new-azvm scripts

Commit:
Updating the new-azvm doc to the new stuff

HeidiSteen committed:
security doc staging

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
TOC edits, headings

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

Q

:

:

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
art, edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
art, edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
art, edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
finished draft

HeidiSteen committed:
art links

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
typos

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
typo

Commit:
Azure PS version update

AjayKumar-N committed:
(ForumsTeam) Under the section "Getting Started" the link "get the appropriate SDK" is broken

In the referenced document: https://docs.microsoft.com/en-us/azure/fundamentals-introduction-to-azure#commerce
The section  "Getting Started" the link "get the appropriate SDK" is broken, request to replace that with the appropriate link.
https://azure.microsoft.com/en-us/downloads/

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

Commit:
Move VM freshness

Commit:
acrolinx

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
TOC edits, link changes, intro section revised

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
restored metadata values in the security filter docs; we should preserve original strings

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
acro edits, typos

Commit:
PS version

Commit:
PS version

Commit:
Spelling

Commit:
pre-check issues

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
edits

HeidiSteen committed:
typo

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
typo

chrissie926 committed:
add quote

chrissie926 committed:
add includes

chrissie926 committed:
Merge branch 'master' into master

belmaiastar committed:
Assign AAD user role on new Azure portal

belmaiastar committed:
Remove unused pictures

belmaiastar committed:
Add reference to the media

Commit:
testingssyntax

belmaiastar committed:
Update reply url info

belmaiastar committed:
Update Azure portal link

Commit:
clarification

belmaiastar committed:
Fix broken link

belmaiastar committed:
Fix typo

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
Security overview doc, soc 2 compliance, TOC changes

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-security

HeidiSteen committed:
acrolinx error

billmath committed:
updating

CarlRabeler committed:
cancel operations - take 1

CarlRabeler committed:
take 2

CarlRabeler committed:
acro

CarlRabeler committed:
png

CarlRabeler committed:
new red box

billmath committed:
updating

mimig1 committed:
Cosmos DB: Image updates #6.

iainfoulds committed:
Freshness updates and revisions

billmath committed:
fixing title

Commit:
Formatting

bandersmsft committed:
cost mgt - updated old link

chrissie926 committed:
fix quote style

chrissie926 committed:
fix types

chrissie926 committed:
Merge branch 'master' of https://github.com/chrissie926/azure-docs-pr

Juliako committed:
custom domain updates

iainfoulds committed:
Additional freshness updates

billmath committed:
updating tou2

Ckarst committed:
Update sql-data-warehouse-load-from-azure-data-lake-store.md

simplified language

bandersmsft committed:
updated faq

bandersmsft committed:
fixed formatting

billmath committed:
updating image

Ja-Dunn committed:
Merge pull request #30068 from mimig1/12142017images6

Cosmos DB: Image updates #6.

PRMerger10 committed:
Merge pull request #30085 from Ckarst/patch-17

Update sql-data-warehouse-load-from-azure-data-lake-store.md

cwatson-cat committed:
Updated ms.author on Backup topics so I can retire them.

mimig1 committed:
Cosmos DB: Intro page edits

belmaiastar committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aad_new_portal

raiye committed:
Update cloud-services-guestos-update-matrix.md

Juliako committed:
custom domain updates

iainfoulds committed:
Additional doc updates

raiye committed:
Update cloud-services-guestos-update-matrix.md

iainfoulds committed:
Acrolinx updates

billmath committed:
updating image

MicrosoftGuyJFlo committed:
Callout of API date limits

euangms committed:
Remove reference to Azure DW support

PRMerger17 committed:
Merge pull request #30071 from iainfoulds/vmfreshness121417-batch1

VM freshness updates

Ja-Dunn committed:
Merge pull request #29957 from linggengmsft/aad_new_portal

Assign AAD user role on Azure new portal

Commit:
Merge pull request #30092 from raiye/patch-1

Update cloud-services-guestos-update-matrix.md

kevinvngo committed:
Update language to data warehouse

Commit:
Merge pull request #30095 from euangms/master

Remove reference to Azure DW support

PRMerger16 committed:
Merge pull request #30094 from MicrosoftGuyJFlo/SSPRReportingNote

Callout of API date limits

Ja-Dunn committed:
Merge pull request #29920 from chrissie926/master

add quote

kevinvngo committed:
updated server view

belmaiastar committed:
Update AAD screenshots

kevinvngo committed:
Update image to connect

kevinvngo committed:
Updated image

billmath committed:
fixing indent

rwike77 committed:
fixing broken link

Ja-Dunn committed:
Merge pull request #30098 from linggengmsft/aad_new_portal_picture

Update AAD screenshots to the standard

PRMerger10 committed:
Merge pull request #29768 from HemantMahawar/pscloudshell-azurermversion

Azure PS version update

Commit:
date

PRMerger12 committed:
Merge pull request #30100 from rwike77/freshness

fixing broken link

SoubhagyaDash committed:
Merge pull request #5 from MicrosoftDocs/master

sync from upstream

jeffpatt24 committed:
Update storage-sync-files-scale-targets.md

Updated sync group limits

JeffGoldner committed:
Update list of extensions

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

v-thepet committed:
fix broken link

SoubhagyaDash committed:
Availability qna updated for intermittent failures issue

bandersmsft committed:
fixed link

GitHubber17 committed:
Merge pull request #29783 from cynthn/update-demo

Updating the new-azvm doc to the new stuff

GitHubber17 committed:
Merge pull request #29784 from cynthn/remove-auto

Removing new-azvm scripts

GitHubber17 committed:
Merge pull request #29785 from cynthn/ps-quick

Ps quickstart with new-asurerrmvm

bandersmsft committed:
fixed link

GitHubber17 committed:
Merge pull request #30099 from kevinvngo/patch-19

Update - connnect to SQL Data Warehouse

GitHubber17 committed:
Merge pull request #30104 from jeffpatt24/patch-8

Update storage-sync-files-scale-targets.md

v-thepet committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into newfix

GitHubber17 committed:
Merge pull request #30077 from bandersmsft/master

cost mgt - updated old link

v-thepet committed:
another broken link

GitHubber17 committed:
Merge pull request #30081 from Juliako/APIMUpdate

custom domain updates

GitHubber17 committed:
Merge pull request #30106 from JeffGoldner/master

Added some more extensions

Commit:
Merge pull request #30107 from v-thepet/newfix

fix broken link

PRMerger18 committed:
Merge pull request #30113 from SoubhagyaDash/master

Add documentation on known causes of intermittent failures

malvenko committed:
Corrected the product name in Samples

Corrected the product name in Samples (Azure Monitor -> Log Analytics)

PRMerger19 committed:
Merge pull request #30118 from malvenko/patch-1

Corrected the product name in Samples

mayanknayar committed:
MVC changes for ASR networking

MVC updates for ASR networking documentation

mayanknayar committed:
MVC change for AD with Site Recovery

MVC change for AD with Site Recovery

linda33wj committed:
Add ADLS troubleshooting tips

linda33wj committed:
Refine sub title

Commit:
Merge pull request #30122 from linda33wj/master

Add ADLS troubleshooting tips

carolinacmoravia committed:
Update tutorial-control-flow.md

PRMerger16 committed:
Merge pull request #26488 from carolinacmoravia/patch-414

M12648: Case of ambiguity:  "activities on which email address"

PRMerger14 committed:
Merge pull request #29824 from AjayKumar-N/patch-12

(ForumsTeam) Under the section "Getting Started" the link "get the ap…

mimig1 committed:
additional updates

Commit:
fixing bold

nsoneji committed:
Updated worload and corrected Azure site limit for H2A

itechedit committed:
Merge pull request #30040 from billmath/tou2

updating

itechedit committed:
Merge pull request #30115 from mimig1/12142017intro

Cosmos DB: Intro page edits

itechedit committed:
Merge pull request #30120 from mayanknayar/master

MVC changes for ASR networking

CarlRabeler committed:
Update sql-database-monitor-and-scale-database-powershell.md

nsoneji committed:
minor update

CarlRabeler committed:
Update sql-database-single-database-resources.md

nsoneji committed:
correct RDS link

itechedit committed:
Update virtual-machines-common-move-vm.md

itechedit committed:
Merge pull request #30090 from cwatson-cat/12-14-BackuptopicRetire1

Updated ms.author on Backup topics so I can retire them.

angusrtaylor committed:
modified required dependencies

angusrtaylor committed:
edited confusing data description

Lisaco88 committed:
Merge pull request #29984 from HeidiSteen/heidist-security

 Security overview doc, soc 2 compliance, TOC changes

Lisaco88 committed:
Merge pull request #30051 from CarlRabeler/ayo

cancel operations - take 1

itechedit committed:
Merge pull request #29833 from cynthn/move-fresh

Move VM freshness

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #30129 from nsoneji/master

Updated worload and corrected Azure site limit for H2A

v-alje committed:
Merge pull request #30132 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #30133 from angusrtaylor/tutorialdocsupdate

Tutorialdocsupdate
